pyproject.toml

```diff
@@ -32,13 +32,19 @@ dependencies = [
 	'importlib_metadata >= 3.7, <5; python_version < "3.8"',
 	"mapbox_earcut >= 0.12.10",
 	"matplotlib ~= 3.2",
-	"numpy ~= 1.22",
+	"networkx ~= 3.0",
+	"numpy ~= 1.21",
```

Comment:
Any reason to roll back the required `numpy` version? I think I increased it because of some security vulnerability.

Reply:
Hmm must have been a merge bug. It's 1.24 now which is the most recent version so we should be OK.